### PR TITLE
update readme of EAST for MindSpore Lite inference guideline

### PR DESCRIPTION
**Pull request by tonytonglt**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: configs/det/east/README_CN.md [hunk 1]
```diff
@@ -146,6 +146,48 @@ mpirun --allow-run-as-root -n 8 python tools/train.py --config configs/det/east/
 python tools/eval.py --config configs/det/east/east_r50_icdar15.yaml
 ```
 
+### 3.6 MindSpore Lite 推理
+
+在进行推理前，请确保您已经完成[推理环境搭建](../../../docs/cn/inference/environment_cn.md)。完成环境准备后，请先下载[MindIR模型](https://download.mindspore.cn/toolkits/mindocr/east/east_resnet50_ic15-7262e359-5f05cd42.mindir)文件或使用以下命令将已训练完成的ckpt导出成MindIR文件:
```
Comment (liangxhao) [inline configs/det/east/README_CN.md:151]: 先链接指向推理教程的入口文件inference_tutorial_cn.md，再依次讲步骤

Comment (HaoyangLee) [inline configs/det/east/README_CN.md:151]:
```suggestion
在进行推理前，请确保您已经完成[推理环境搭建](../../../docs/cn/inference/environment_cn.md)。完成环境准备后，你需要使用模型的MindIR文件进行推理。获得MindIR文件有如下两种方式：
- （1）下载本仓库已训练好的MindIR模型，请见[2. 实验结果](#2-实验结果)章节
- （2）使用[`tools/export.py`](../../tools/export.py)将你训练完成的ckpt导出成MindIR文件，命令如下：
```

File: configs/det/east/README_CN.md [hunk 2]
```diff
@@ -146,6 +146,48 @@ mpirun --allow-run-as-root -n 8 python tools/train.py --config configs/det/east/
...
+
+``` shell
+python tools/export.py --model_name east_resnet50 --data_shape 720 1280 --local_ckpt_path /path/to/local_ckpt.ckpt
```
Comment (HaoyangLee) [inline configs/det/east/README_CN.md:154]: 补充支持通过传入yaml文件进行export的用法，参考https://github.com/mindspore-lab/mindocr/blob/6ee21a85590890578278f919611db14b7b136a58/tools/export.py#L16

File: configs/det/east/README.md [hunk 3]
```diff
@@ -150,6 +151,48 @@ To evaluate the accuracy of the trained model, you can use `eval.py`. Please set
...
+
+Please refer to [Model Conversion](../../../docs/en/inference/convert_tutorial_en.md#1-mindocr-models),
+and use the `converter_lite` tool for offline conversion of the MindIR file, where the `input_shape` in `configFile` needs to be filled in with the value from MindIR export,
+as mentioned above (1, 3, 720, 1280), and the format is NCHW.
+
+- Inference
+
+Assuming that you obtain output.mindir after model conversion, go to the `deploy/py_infer` directory, and use the following command for inference:
+
+Use the converter_lite tool to convert this MindIR file to a MindIR model supported by MindSpore Lite:
```
Comment (liangxhao) [inline configs/det/east/README.md:184]: 多写了，这在Model Conversion已经做过了

File: configs/det/east/README.md [hunk 4]
```diff
@@ -44,6 +44,7 @@ After determining the location and size of the text region, EAST further classif
 #### Notes：
 - Context: Training context denoted as {device}x{pieces}-{MS version}{MS mode}, where mindspore mode can be G - graph mode or F - pynative mode with ms function. For example, D910x8-G is for training on 8 pieces of Ascend 910 NPU using graph mode.
 - The training time of EAST is highly affected by data processing and varies on different machines。
```
Comment (liangxhao) [inline configs/det/east/README.md:46]: 句尾英文句号

File: configs/det/east/README.md (hunk 3 above)
Comment (tonytonglt) [reply configs/det/east/README.md]: 已删除

File: configs/det/east/README.md (hunk 4 above)
Comment (tonytonglt) [reply configs/det/east/README.md]: 已修改

File: configs/det/east/README_CN.md (hunk 1 above)
Comment (tonytonglt) [reply configs/det/east/README_CN.md]: 已修改

[the remark this answers is not in the document]
Comment (tonytonglt) [reply configs/det/east/README_CN.md]: 已修改

File: configs/det/east/README_CN.md [hunk 5]
```diff
@@ -146,6 +146,48 @@ mpirun --allow-run-as-root -n 8 python tools/train.py --config configs/det/east/
...
+    --device=Ascend \
+    --modelFile=east_resnet50.mindir \
+    --outputFile=output \
+    --configFile=config.txt
+```
+上述命令将生成output.om以及output.mindir模型文件。其中，config.txt文件配置如下：
+```
+ [ascend_context]
+ input_format=NCHW
+ input_shape=x:[1,3,720,1280]
```
[the remark this answers is not in the document]
Comment (tonytonglt) [reply configs/det/east/README_CN.md]: 已增加

File: configs/det/east/README_CN.md (hunk 1 above)
Comment (tonytonglt) [reply configs/det/east/README_CN.md]: 已修改

File: configs/det/east/README_CN.md (hunk 2 above)
Comment (tonytonglt) [reply configs/det/east/README_CN.md:159]: 已修改

**Review by liangxhao — APPROVED** [review]

**Review by HaoyangLee — APPROVED** [review]

**Pull request merged**